packages/database/src/core/migrations/20250505235002-addSetUpdatedAtSyncTickFunction-modifies-schema.js

```diff
@@ -32,14 +32,16 @@ exports.up = function (db) {
         IF NEW.updated_at_sync_tick = -1 THEN
           NEW.updated_at_sync_tick := -999;
         ELSE
-          -- take an advisory lock on the current sync tick (if one doesn't already exist), to
+           -- First get the current sync tick
+          SELECT value FROM local_system_facts WHERE key = 'currentSyncTick' INTO current_tick;
```

Comment:
I think you need to declare `current_tick`
https://github.com/beyondessential/tamanu/pull/7540/files/717bf307dd1cece7328525f0c56f2b81e0306765..65b2fe63855c7883c007b642cb58f9fa0922a34a#diff-95aee4796ccdf9e693f257ef44d3ecc9e882c38a7d3bddda81e4f3c264c2ccfbR11